### DISCUSSION TITLE
艾尔登法环 分段压缩包打不开，下载3次了

### DISCUSSION THREAD

**Discussion by bluesiboy** [Gamuxorg/bbs#392] [建议和闲聊]
39279ba42c13681e606ca0de35be6fb7  eldenring_1.03.2_amd64.appimage.zip.001
965a5637783b151a6e4859a465d14c0b  eldenring_1.03.2_amd64.appimage.zip.002
af5ee9c92b0b4912c2be32982b822a4b  eldenring_1.03.2_amd64.appimage.zip.003
651955c2e03a6e479296ebea0a11a505  eldenring_1.03.2_amd64.appimage.zip.004
29f13a38d0aee81bdf612c104542ad38  eldenring_1.03.2_amd64.appimage.zip.005
f1dd1a92b96f863767ff40e4a18e396b  eldenring_1.03.2_amd64.appimage.zip.006
c80dcc8a22ea9706c3f88952461a9838  eldenring_1.03.2_amd64.appimage.zip.007
7a92abb3062cc06687ba84f46eb5709a  eldenring_1.03.2_amd64.appimage.zip.008
d7bf038b87f8bfc30507c4778135f774  eldenring_1.03.2_amd64.appimage.zip.009
9ed04eb4049daa868cd426597bb7b224  eldenring_1.03.2_amd64.appimage.zip.010
377dd7dd8640d315d8405e70add06ecb  eldenring_1.03.2_amd64.appimage.zip.011
9d224f4ae9400070f2828cc8dce77dfd  eldenring_1.03.2_amd64.appimage.zip.012
aa1e791586575fd56c4abd3c99a78076  eldenring_1.03.2_amd64.appimage.zip.013
eeba34ab75e0ce1a273fa0bca458d399  eldenring_1.03.2_amd64.appimage.zip.014

**Comment by shouhuanxiaoji** [comment]
收到，尽快验证下
